content/finalist/ttjc@finalist@Shivraj97/index.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Shivraj97
+type: "finalist"
+name: "Shivraj Nag"
+portfolio: "https://shivrajnag.netlify.app/"
+github-link: "https://github.com/Shivraj97"
+linkedin-link: "https://www.linkedin.com/in/shivraj-n-5150a1b2/"
+looking-for: "job"
```

Comment:
can you insert image link here, after line 8 as follows:

image-link: "link goes here"

Reply:
ok